### PR TITLE
add Confirm links to scheduled tasks

### PR DESCRIPTION
**Pull request by ericcf**
[#114731149]

### CODE REVIEW

File: app/models/nurse_task.rb [hunk 1]
```diff
@@ -47,6 +47,10 @@ def resolve
     update status: STATUSES.resolved
   end
 
+  def target
```
Comment (michaelwehrley) [inline app/models/nurse_task.rb]: Did you think about having the parent class have a method such as...
```ruby
private

def symbolize(klass)
  klass.to_s.underscore.to_sym
end
```
and then in the other classes:
```ruby
#app/models/tasks/call_to_schedule_final_appointment.rb
def target
  symbolize(::CallToScheduleFinalAppointment)
end
```
```ruby
#app/models/tasks/confirmation_call.rb
def target
  symbolize(FirstContact)
end
```
It just seems like a lot of duplicated methods.

File: app/views/tasks/index.html.erb [hunk 2]
```diff
@@ -21,15 +21,21 @@
...
+            <%= button_to "Mark as resolved",
+                          resolve_participant_task_path(participant_id: @tasks.id, id: task.id),
+                          { method: :put, class: "btn btn-default" } %>
+            <%= button_to "Contact Supervisor",
+                          notify_supervisor_participant_task_path(participant_id: @tasks.id, id: task.id),
+                          { class: "btn btn-default", data: { confirm: "are you sure you want to notify the supervisor that you need help?" } } %>
+            <p>last supervisor contact sent <%= l @tasks.latest_notification_at, format: :long %></p>
+          <% else %>
+            <%= link_to "Confirm",
+                new_polymorphic_path([:participant, task.target], { participant_id: @tasks.id }),
```
Comment (michaelwehrley) [inline app/views/tasks/index.html.erb]: :smile_cat:

File: app/models/nurse_task.rb (hunk 1 above)
Comment (ericcf) [inline app/models/nurse_task.rb]: Yeah, that seems cleaner to me!

File: spec/views/tasks/index.html.erb_spec.rb [hunk 3]
```diff
@@ -106,4 +106,24 @@ def stub_alert_tasks
...
     end
   end
+
+  context "for scheduled tasks" do
+    def stub_scheduled_tasks
+      task = instance_double(NurseTask,
+                             active?: true,
+                             scheduled_at: Time.zone.now,
+                             target: :first_contact,
+                             alert?: false).as_null_object
```
Comment (michaelwehrley) [inline spec/views/tasks/index.html.erb_spec.rb]: Nice additional spec.

**Comment by michaelwehrley** [comment]
:+1:

**Pull request merged**